gcp/appengine/blog/content/posts/introducing-broad-c-c++-support/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Introducing broad C/C++ vulnerability management support"
+date: 2023-11-03T00:10:00Z
+draft: false
+author: Andrew Pollock and Oliver Chang
+---
+OSV is committed to bringing our users comprehensive, accurate and timely open source vulnerability information. Over the last year, we’ve released a number of new features in pursuit of this goal including:
+- [OSV-Scanner’s call graph analysis for Go and Rust](https://google.github.io/osv-scanner/experimental/#scanning-with-call-analysis)
+- Adding six new ecosystems to the database
+- [The determineversion API](https://osv.dev/blog/posts/using-the-determineversion-api/), which expanded access to C/C++ vulnerabilities for OSS-Fuzz projects 
+
+Today we are announcing that OSV advisories now include vulnerable commit ranges. **Vulnerable commit ranges, along with the previously announced experimental determineversion API, will enable vulnerability management for software with C and C++ dependencies, which has been one of the last gaps in coverage in OSV.dev’s database. Additionally OSV-Scanner is now compatible with C and C++ projects. **
+<!--more-->
+
```

Comment:
I think you could add a heading before this paragraph, to make the announcement above stand out and guide the reader in understanding what these next two paragraphs are saying.